runtime/doc/lsp.txt

```diff
@@ -2159,6 +2160,30 @@ is_enabled({filter})                         *vim.lsp.inlay_hint.is_enabled()*
         (`boolean`)
 
 
+==============================================================================
+Lua module: vim.lsp.inline_completion                  *lsp-inline_completion*
+
+accept({bufnr})                           *vim.lsp.inline_completion.accept()*
```

Comment:
👍 select() already has 2 other overloaded meanings, so `accept()` seems like the right choice here.